guides/09_Amazon_Voice_Focus.md

```diff
@@ -369,6 +369,19 @@ await deviceController.startAudioInput(this.voiceFocusDevice);
 console.log('Amazon Voice Focus switched to new device', newDevice);
 ```
 
+## Destroying Amazon Voice Focus worker thread from your application
+
+In some cases, you might want to completely stop the worker thread that runs the `VoiceFocusDeviceTransformer`. In that case, you can use the `destroyVoiceFocus` method to completely remove the voice focus worker thread. After destroying the `VoiceFocusDeviceTransformer`, you'd now have to create a new one. This might be useful if you intend to completely remove Voice Focus from your application at some point. If there was a `VoiceFocusTransformerDevice` associated with the destroyed transformer, that `VoiceFocusTransformerDevice` will stop working, as the transformer has been destroyed. You can also call `VoiceFocusTransformerDevice.stop()` to stop the worker thread, but that assumes that you've already created a `VoiceFocusTransformerDevice`. If you havent created a `VoiceFocusTransformerDevice` already, you should use `VoiceFocusDeviceTransformer.destroyVoiceFocus` to destroy the Voice Focus worker thread.
```

Comment:
- `the voice focus worker thread` => `the Voice Focus worker thread
- `havent` => `haven't`